### PR TITLE
Fix docker deploys

### PR DESCRIPTION
**Pull request by jpalvarezl**

### CODE REVIEW

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -151,7 +151,9 @@ jobs:
 
       - name: Deploy interstellar
         if: github.ref == 'refs/heads/interstellar'
-        run: bash scripts/deploy_docker.sh interstellar
+        run: |
+          bash scripts/deploy_docker.sh interstellar
```
Comment (rmeissner) [inline .github/workflows/rust.yml:155]: new line indicator missing at the end

File: .github/workflows/rust.yml [hunk 2]
```diff
@@ -151,7 +151,9 @@ jobs:
...
+          curl -H "Content-Type: application/json" -X POST -d '{"push_data": {"tag": "staging" }}' ${{ AUTODEPLOY_URL }}
```
Comment (rmeissner) [inline .github/workflows/rust.yml:156]: is `staging` the correct tag here?

Comment (rmeissner) [reply .github/workflows/rust.yml]: shouldn't this be `interstellar`?

Comment (jpalvarezl) [reply .github/workflows/rust.yml]: wasn't this supposed to be `staging` for the deployment prefix?

**Review by rmeissner — APPROVED** [review]

**Pull request merged**